libs/heattracesidesheet/src/lib/ui-sidesheet/heatedTagsColumns.tsx

```diff
@@ -0,0 +1,51 @@
+import { ColDef } from '@equinor/workspace-ag-grid';
+import { ICellRendererProps } from '@equinor/workspace-fusion/grid';
```

Comment:
Import line 1 and 2 should be able to import from same package

Reply:
Fixed.